control/server.py

```diff
@@ -83,7 +83,7 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.server.stop(None)
 
         self.logger.info("Exiting the gateway process.")
-        return True
+        return False
```

Comment:
I think implicit return is the usual style for context manager exit methods.  That is what `OmapGatewayState` does, for example, so I'd just drop the return statement here to be consistent.

Reply:
You're completely right. I initially removed it and later decided to leave it to make it explicit (I also thought about adding a cautionary comment pointing to this very same issue). Given the 'catastrophic' effects of this one-liner, I checked if some static analyzer would catch it (since they immediately report `except: pass` as dangerous), but to my surprise neither pylint nor flake8 caught it...

What do you think?

Reply:
> What do you think?

The only reason I commented is that I don't remember seeing an explicit `return False` in single-return (where there is no desire to swallow the exception in some particular case) exit methods that much.  But then I don't read a lot of Python so I'll leave it up you.  It's a really minor nit.